### PR TITLE
Dont allow skewScale 0 

### PR DESCRIPTION
**Pull request by 0xjocke**

### CODE REVIEW

**Comment by davidvuong** [comment]
Code looks fine but consider writing a test with zero skewScale. Also, failing test

> LiquidationModule "before each" hook: restore for "should remove all margin collateral on flag"

🤔

**Comment by 0xjocke** [comment]
> Code looks fine but consider writing a test with zero skewScale. Also, failing test
> 
> > LiquidationModule "before each" hook: restore for "should remove all margin collateral on flag"
> 
> 🤔

Yep will do, had problems running test at all yesterday. Main is also failing on this one. I just tried nuking cache on main an reran. But no success so far

File: markets/bfp-market/contracts/modules/MarketConfigurationModule.sol [hunk 1]
```diff
@@ -53,6 +54,10 @@ contract MarketConfigurationModule is IMarketConfigurationModule {
         // Only allow an existing per market to be configurable. Ensure it's first created then configure.
         PerpMarket.exists(marketId);
 
+        if (data.skewScale == 0) {
+            revert ErrorUtil.ZeroAmount();
```
Comment (davidvuong) [inline markets/bfp-market/contracts/modules/MarketConfigurationModule.sol:58]: I want to add more configuration validation checks here. Can we use a `error InvalidParameter(string parameter, string reason);` here instead?

Comment (0xjocke) [reply markets/bfp-market/contracts/modules/MarketConfigurationModule.sol:58]: Yep cool I was thinking about that too

Comment (0xjocke) [reply markets/bfp-market/contracts/modules/MarketConfigurationModule.sol]: Done!

**Review by davidvuong — APPROVED** [review]

**Pull request merged**